front/cypress/e2e/idea_custom_field.cy.ts

```diff
@@ -0,0 +1,163 @@
+import { randomString } from '../support/commands';
+
+describe('Idea custom field', () => {
+  const ideaTitle = randomString(40);
+  const ideaContent = randomString(60);
+  const initialNumberOfBenches = '15';
+  const editedNumberOfBenches = '28';
+
+  let projectId: string;
+  let projectSlug: string;
+
+  beforeEach(() => {
+    cy.getProjectBySlug('custom-idea-fields-project').then((project) => {
+      projectId = project.body.data.id;
+      projectSlug = project.body.data.attributes.slug;
+    });
+
+    cy.setAdminLoginCookie();
+  });
+
+  it('exists on the Input form page', () => {
+    cy.visit(`admin/projects/${projectId}/ideaform`);
+    cy.acceptCookies();
+
+    // Check to see that custom field exists on input form
+    cy.get('.e2e-number_of_benches-setting-collapsed').should('exist');
+  });
+
+  /* TODO: Change test to verify custom value on idea show page
+   * This test navigates to the edit page again to verify that the custom idea field is there.
+   * This should be changed once we show the custom idea field in the idea show page.
+   */
+  it('can be created on an idea and values edited', () => {
```

Comment:
The other example reads `of type string can be created on an idea and values edited`, so please indicate `of type number` here as well.